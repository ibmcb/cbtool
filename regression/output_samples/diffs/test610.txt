 status: VMC D30F1B96-9225-51C9-95FF-3355AD659B11 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 671172C3-2E80-5782-AB0C-C348A586106A was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 96342978-982B-5D60-9FF3-0C0B3F9C41A9 was successfully unregistered on SimCloud "TESTCLOUD".
