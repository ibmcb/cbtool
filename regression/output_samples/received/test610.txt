 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Removing all SVM objects attached to this experiment.
 status: Removing all VMC objects attached to this experiment.
 status: VMC D30F1B96-9225-51C9-95FF-3355AD659B11 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 671172C3-2E80-5782-AB0C-C348A586106A was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 96342978-982B-5D60-9FF3-0C0B3F9C41A9 was successfully unregistered on SimCloud "TESTCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Disassociating default cloud: TESTCLOUD
Cloud TESTCLOUD was successfully detached from this experiment.
