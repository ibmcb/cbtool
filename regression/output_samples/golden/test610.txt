 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Removing all SVM objects attached to this experiment.
 status: Removing all VMC objects attached to this experiment.
 status: VMC 35310FEB-1E59-53F1-A37C-0E4387DFF893 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC DAEF899C-8C95-5B76-BE3F-58349C261E66 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 8E18B916-CC95-566E-887E-3032BA7F835A was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 3076C295-3106-5ADC-8C71-D21083902D7A was successfully unregistered on SimCloud "TESTCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Disassociating default cloud: TESTCLOUD
Cloud TESTCLOUD was successfully detached from this experiment.
